appstore_tools/actions/publish.py

```diff
@@ -293,13 +314,44 @@ def publish_preview(
         colorama.Fore.CYAN,
         "commiting upload",
     )
+
+    preview_frame_time_code_filepath = os.path.join(
```

Comment:
In the case that we need to add other configurable values per preview file in the future, let's decide if each would be in their own file or all in one file as named values.

Option 1:
- example.avi
- example_preview_frame_time_code.txt
- example_some_future_option.txt

Option 2:
- example.avi
- example.txt
  ```
  # contents of example.txt
  preview_frame_time_code=00:00:00:01
  some_future_option=true
  ```

Reply:
I probably lean more towards option 2.